it.hjson

```diff
@@ -58,17 +58,17 @@
   "generic.buttons": {
     "whatIsThis": "❓ Cos'è questo?"
     "whatIsThisShort": "❓ Cosa?"
-    "imSorry": "😭 Mi dispiace!"
+    "imSorry": "😭  Mi dispiace!"
 
     "joinChannel": "👋 Iscriviti al canale"
-    "tapToVerify": "🤖 Toccare per verificare 🤖"
+    "tapToVerify": "🤖 Tocca per iniziare la verifica 🤖"

```

Comment:
Is this the shortest version possible?

It's important that we make sure this never gets cropped in the inline button, even on mobile devices